docs/content/en/blog/articles/dev-config/index.md

```diff
@@ -0,0 +1,321 @@
+* Run the whole stack, if possible, with at least three Cassandra nodes and one Stargate node. K8ssandra ships with the following open source components:
+    * [Apache Cassandra&reg;](http://cassandra.apache.org/) 
+    * [Stargate](https://stargate.io/) : API framework and data gateway (CQL, REST, GraphQL)
+    * [Medusa for Apache Cassandra&reg;](https://github.com/thelastpickle/cassandra-medusa) : Backup and restore tool for Cassandra
+    * [Reaper for Apache Cassandra&reg;](http://cassandra-reaper.io/) : Repair orchestration tool for Cassandra
+    * [Metrics Collector for Apache Cassandra](https://github.com/datastax/metric-collector-for-apache-cassandra) : Metric collection and Dashboards for Apache Cassandra (2.2, 3.0, 3.11, 4.0) clusters.
+    * [Management API for Apache Cassandra](https://github.com/datastax/management-api-for-apache-cassandra) : Secure Management Sidecar for Apache Cassandra
+    * [Cass Operator](https://github.com/datastax/cass-operator) : Kubernetes Operator for Apache Cassandra
+    * [Medusa Operator](https://github.com/k8ssandra/medusa-operator) : Kubernetes Operator for Medusa
+    * [Reaper Operator](https://github.com/k8ssandra/reaper-operator) : Kubernetes Operator for Reaper
+    * [kube-prometheus-stack](https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack) chart:
+        * [Prometheus](https://prometheus.io/) : Monitoring system & time series database
+        * [Grafana](https://grafana.com/) : Fully composable observability stack
+* Achieve reasonable startup times
+* Specify a dev setup stable enough to sustain moderate workloads (50 to 100 ops/s)
+* Come up with some minimum requirements and recommended K8ssandra settings
+
+# Using the right settings
+Cassandra can run with fairly limited resources as long as you don’t put too much pressure on it. For example, for the Reaper project, we run our integration tests with [CCM (Cassandra Cluster Manager)](https://github.com/riptano/ccm), configured at [256MB of heap size](https://github.com/thelastpickle/cassandra-reaper/blob/master/.github/scripts/configure-ccm.sh#L22). This allows the JVM to allocate an additional 256MB of off heap memory, allowing Cassandra to use up to 512MB of RAM.
+
+If we want to run K8ssandra with limited resources, we’ll need to set these appropriately in our Helm values files.
+
+## Setting heap sizes in K8ssandra
+The K8ssandra Helm charts allow us to set heap sizes for both the Cassandra and Stargate pods separately.
+
+### Cassandra
+For Cassandra, the heap and new gen sizes can be set at the cluster level, or at the datacenter level (K8ssandra will support multi DC deployments in a future release):
+
+```
+cassandra:
+  version: "3.11.10"
+  ... 
+  ...
+  # Cluster level heap settings
+  heap: {}
+   #size:
+   #newGenSize:
+
+  datacenters:
+  - name: dc1
+    size: 3
+    ... 
+    ... 
+    # Datacenter level heap settings
+    heap: {}
+      #size:
+      #newGenSize:
```

Comment:
Do we want to set `newGenSize` if `G1` is in use? It was my understanding that the parameter should be left blank. See https://docs.datastax.com/en/dse/6.0/dse-admin/datastax_enterprise/operations/opsTuningGcAbout.html#G1advantages

(Note if we use CMS my comment may be safely ignored).

Reply:
thoughts @adejanovski ?